src/app/services/files.js

```diff
@@ -278,7 +279,10 @@ module.exports = (Model, App) => {
   };
 
   const MoveFile = async (user, fileId, destination) => {
-    const file = await Model.file.findOne({ where: { fileId: { [Op.eq]: fileId } }, userId: user.id });
+    const file = await Model.file.findOne({
+      where: { fileId: { [Op.eq]: fileId }, status: { [Op.not]: 'DELETED' } },
```

Comment:
Change is here. Other changes are just the linter (I tried with 3 linters and they all tried to format it)

Reply:
Nice catch!